frontend/form-builder/src/components/Toolbar/Toolbar.module.css

```diff
@@ -0,0 +1,11 @@
+.toolbar {
+    ul {
+        list-style: none;
+        padding: 0;
+        text-align: center;
+    }
+
+    li {
+        margin-bottom: var(--fds-spacing-4);
+    }
```

Comment:
Her burde vi også bruke klassenavn istedenfor direkte native elementer. 😊

Reply:
Hvorfor ikke bruke native elementer her, når CSS Modules innkapsler stilen innenfor komponenten?

Reply:
Det er egentlig bare for lesbarheten, jeg skjønner ikke hva jeg ser stilen til. Et klassenavn kan gi en lesbar kontekst av hva li er for noe i denne konteksten.